pkg/config/node.go

```diff
@@ -367,10 +368,24 @@ func getNodeIpForRequestedIpStack(node v1.Node, filterIps []string, machineNetwo
 		log.Debugf("For node %s can't find address using NodeInternalIP. Fallback to OVN annotation.", node.Name)
 
 		var ovnHostAddresses []string
-		if err := json.Unmarshal([]byte(node.Annotations["k8s.ovn.org/host-addresses"]), &ovnHostAddresses); err != nil {
+		var tmp []string
+
+		err := json.Unmarshal([]byte(node.Annotations["k8s.ovn.org/host-cidrs"]), &tmp)
```

Comment:
I'm not sure how versioning works, is this going to work with upgrades?  4.13 to 4.14 maybe.  I don't know exactly when the annotation changed.

Reply:
ah, I didn't have it on side-by-side diff.  🤣